tabbycat/utils/tables.py

```diff
@@ -915,6 +915,24 @@ def add_debate_results_columns(self, debates):
                 else:
                     cell = self._result_cell_class_four(debateteam.points, cell)
 
+                if iron and (debateteam.iron > 0 or debateteam.iron_prev > 0):
+                    cell['text'] = "🗣️" + cell['text']
```

Comment:
I might want to change the emoji to be a magnet once it is widely supported. The magnet has some red colouring which is good to gain attention, and it has a connotation to iron. While "iron" may be regional, a person speaking is debating in a nutshell, and does not convey this sense of missing a team member (no emoji adequately represents it universally...)

Reply:
Sounds good, the magnet to signify iron is nice. If the button text stays referencing 'iron' it would be good to tie them together.

We could also potentially use the internal terminology 'ghost' which would then mean 👻 could be dropped in.